spec/unit/installer/analyzer_spec.rb

```diff
@@ -1122,17 +1124,23 @@ module Pod
 
           result.targets.flat_map { |at| at.pod_targets.map { |pt| "#{at.name}/#{pt.name}" } }.sort.should == [
             'Pods-Sample Extensions Project/JSONKit',
+            'Pods-Sample Extensions Project/matryoshka-Bar-Foo',
```

Comment:
this test is for frameworks, I believe there is another one below for static libs does this apply for static libs?